atomic_reactor/plugins/exit_koji_import.py

```diff
@@ -135,6 +136,10 @@ def set_help(self, extra, worker_metadatas):
                         # They are all None
                         extra['image']['help'] = None
 
+    def set_brew_versions(self, extra, worker_metadatas):
+        build_id, brew_version = self.workflow.postbuild_results.get(PulpPullPlugin.key)
```

Comment:
To get the value from worker to orchestrator build, you'd need something like this:

In worker build:
1. `pulp_pull` plugin returns the information needed
2. `store_metadata` reads value from `pulp_pull` and stores it as an annotation, say `registry_content_type`

In orchestrator build:
1. `orchestrate_build` plugin additionally reads `registry_content_type` annotation for each worker build and makes it available in `self.workflow.build_result.annotations`
2. `koji_import` reads from `self.workflow.build_result.annotations['registry_content_type']` to get the information needed.